### PR TITLE
Revert "Add composer file"

### PR DESCRIPTION
**Pull request by samuell**
Reverts rdfio/RDFIO#15

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-	"name": "mediawiki/rdfio",
-	"type": "mediawiki-extension",
-	"description": "RDFIO extension for importing data in RDF format to Semantic MediaWiki",
-	"keywords": [
-		"MediaWiki",
-		"Semantic MediaWiki",
-		"RDF",
-		"Linked Data"
-	],
-	"homepage": "<attempt to pull from ExtensionName.php, else blank>",
```
Comment (mwjames) [inline composer.json]: This needs to be cleaned. "<attempt to pull from ExtensionName.php, else blank>",

Comment (mwjames) [inline composer.json]: Please compare with https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/composer.json

**Comment by samuell** [comment]
Indeed, thanks @mwjames !

File: composer.json [hunk 2]
```diff
@@ -1,34 +0,0 @@
-{
-	"name": "mediawiki/rdfio",
```
Comment (mwjames) [inline composer.json]: "mediawiki/rdfio" requires to be registered with https://packagist.org.

**Pull request merged**